src/mudlet.cpp

```diff
@@ -3658,7 +3658,12 @@ void mudlet::slot_connection_dlg_finished(const QString& profile, int historyVer
 
     tempHostQueue.enqueue(pHost);
     tempHostQueue.enqueue(pHost);
-    pHost->connectToServer();
+    if (connect) {
+        pHost->connectToServer();
+    } else {
+        QString infoMsg = tr("[  OK  ]  - Profile \"%1\" loaded in offline mode.").arg(profile);
```

Comment:
May I suggest a second (on screen) line (end the first with a `"\n"`):
```C++
"Use \"Reconnect\" if you later wish to connect to the Game server."
```

Reply:
I don't think it's necessary to spell things out super explicitly to the users. It's kind of obvious that if you load a profile and you want to connect, you can hit the reconnect button :)